modules/mobile/docs/openapi.yaml

```diff
@@ -1985,6 +1985,40 @@ paths:
             application/json:
               schema:
                 $ref: "./schemas/Errors.yml"
+  /v0/messaging/health/messages{id}/attachments/{attachment_id}:
```

Comment:
```suggestion
  /v0/messaging/health/messages/{id}/attachments/{attachment_id}:
```

Reply:
added this suggestion but you'll need to render again 😄

Reply:
Good catch, thank you!